docusaurus/docs/user-docs/settings/introduction.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Introduction to General Settings
+description: Reviewing the audit logs in Strapi.
+sidebar_position: 2
+---
+
+The _Settings_ feature allows an administrator to adjust all ![Settings icon](/img/assets/icons/settings.svg) _Settings_ from the main navigation of the admin panel, which is divided into 4 sub sections:
```

Comment:
```suggestion
The ![Settings icon](/img/assets/icons/settings.svg) _Settings_ section in the main navigation of the admin panel allows an administrator to adjust various settings for the project. It is divided into 4 sub sections:
```

Suggestion of rephrasing because it sounds a bit weird to me to call the "Settings" section a feature. I also think we could go even more quickly to the point with this sentence 😊